src/Picker/ArticlePickerProvider.php

```diff
@@ -82,7 +82,7 @@ protected function getLinkClass()
     /**
      * {@inheritdoc}
      */
-    protected function getRouteParameters(PickerConfig $config)
+    protected function getRouteParameters(PickerConfig $config = null)
```

Comment:
If that was true, then the URL would always link to the selected value 😉 
see https://github.com/contao/core-bundle/pull/984/files#diff-4e7f624198eef84ce3f501384ca6dcc8R135